mock/mock.go

```diff
@@ -527,6 +527,47 @@ func (m *Mock) AssertNotCalled(t TestingT, methodName string, arguments ...inter
 	return true
 }
 
+// IsMethodCallable checking that the method can be called
+// If the method was called more than `Repeatability` return false
+func (m *Mock) IsMethodCallable(t TestingT, methodName string, arguments ...interface{}) bool {
+	if h, ok := t.(tHelper); ok {
+		h.Helper()
+	}
+	m.mutex.Lock()
+	defer m.mutex.Unlock()
+
+	var isCallable = false
+	for _, v := range m.ExpectedCalls {
+		if v.Method != methodName {
+			continue
+		}
+		if len(arguments) != len(v.Arguments) {
+			continue
+		}
+		if v.Repeatability < v.totalCalls {
+			continue
+		}
+		if isArgsEqual(v.Arguments, arguments) {
+			isCallable = true
+			break
+		}
+	}
+	return isCallable
+}
+
+// Compares arguments
```

Comment:
Even though this is on an unexported function (and therefor it won't show up in `godoc`), it's probably better to include the function name in the documentation comment.

Reply:
Go standard is to have the docstring start with the function's name